Zooniverse/environment_mbtiles.yml

```diff
@@ -0,0 +1,36 @@
+name: mbtilesEnv
```

Comment:
Capitalization here `mbtilesEnv` is different from that used in the rules `mbtilesenv`. We should make these consistent even if it works.